strongbox-web-core/src/main/java/org/carlspring/strongbox/jtwig/extensions/HumanReadableSizeConversionFunction.java

```diff
@@ -0,0 +1,58 @@
+package org.carlspring.strongbox.jtwig.extensions;
+
+import java.util.Collection;
+import java.util.Collections;
+
+import org.jtwig.functions.FunctionRequest;
+import org.jtwig.functions.JtwigFunction;
+
+import com.google.common.collect.Lists;
+
+/**
+ * 
+ * @author ankit.tomar
+ *
+ */
+public class HumanReadableSizeConversionFunction implements JtwigFunction
+{
+
+    @Override
+    public String name()
+    {
+        return "humanReadableSize";
+    }
+
+    @Override
+    public Collection<String> aliases()
+    {
+        return Collections.unmodifiableList(Lists.newArrayList("readableSize"));
+    }
+
+    @Override
+    public Object execute(FunctionRequest request)
+    {
+        request.minimumNumberOfArguments(1);
+        request.maximumNumberOfArguments(1);
+        String byteSize = request.getEnvironment().getValueEnvironment().getStringConverter().convert(request.get(0));
+        try
+        {
+            return getHumanReadableSize(Long.parseLong(byteSize));
+        }
+        catch (NumberFormatException nfe)
+        {
+            // Need to return same value of bytes
+        }
+        return byteSize;
+    }
+
+    private String getHumanReadableSize(long bytes)
+    {
+        int unit = 1000;
+        if (bytes < unit)
+            return bytes + " B";
```

Comment:
Cosmetic: Please, always use braces around conditions and loops. Thanks! :)

Reply:
Noted